test-suite-http-server-tck-tomcat/src/test/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "name": "io.micronaut.http.server.tck.tests.BodyTest$Point",
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "io.micronaut.http.server.tck.tests.ConsumesTest$Pojo",
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  }
+]
```

Comment:
it would be better if these existed in the TCK

Reply:
Or should we annotate the TCK with `@Introspected`, the this configuration should not be required right https://github.com/micronaut-projects/micronaut-core/blob/4.0.x/http-server-tck/src/main/java/io/micronaut/http/server/tck/tests/BodyTest.java#L141 ?

Reply:
Yes that is the alternative